ext/webidl/00_webidl.js

```diff
@@ -714,7 +714,7 @@
           throw makeException(
             TypeError,
             `can not be converted to '${name}' because '${key}' is required in '${name}'.`,
-            { ...opts },
+            { context: opts.context, prefix: opts.prefix },
```

Comment:
Does it work to pass simply `opts` instead of creating a new object?

Reply:
It definitely should and there are examples of doing exactly that a few lines earlier, but I'm afraid of making functional changes to the code.

That being said, this whether or not this copying is done cannot be a functional change I don't think. I'll fix this into a simple pass.